services/user_service/test_user_service.py

```diff
@@ -27,18 +47,18 @@ def test_delete_user_by_email():
     assert database_manager.get_user_by_email(user_email)["email"] == user_email
     database_manager.delete_user_by_email(user_email)
     print("STEP 2: After deleting by email user is not more stored")
-    assert database_manager.get_user_by_email(user_email) == []
+    assert database_manager.get_user_by_email(user_email) == None
 
 
 def test_delete_user_by_id():
     print("Running test test_delete_user_by_id")
     user = mocker.get_mocked_users()[0]
     user_id = database_manager.save_user(user)
     print("STEP 1: User has been saved into database")
-    assert database_manager.get_user_by_id(user_id) != []
+    assert database_manager.get_user_by_id(user_id) != None
```

Comment:
Guy from my company said, that with None it is better to use "is not"
Is it really matter?

Reply:
yes, it's more pythonic) Will correct it

Reply:
Also, is None is a bit (~50%) faster than == None

Reply:
https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none